StripeCore/StripeCore/Source/API Bindings/StripeAPI.swift

```diff
@@ -68,9 +68,33 @@ import PassKit
             }
         }
     }
+
+    /// Users that have the Payment Method Cartes Bancaires set to Active
+    /// can enable Cartes Bancaires for Apple Pay by setting this to `YES`.
+    ///
+    /// The default value is NO.
+    @objc public class var cartesBancairesPaymentNetworkSupported: Bool {
```

Comment:
A question for the reviewer:
I see on the [PaymentSheet docs](https://stripe.dev/stripe-ios/stripepaymentsheet/documentation/stripepaymentsheet/additionalenabledapplepaynetworks) that to add an additional network to the Payment Sheet Apple Pay we should use `StripeAPI.additionalEnabledApplePayNetworks`, so I'm assuming this would be enough to support CB Apple Pay on Payment Sheet too?

Reply:
Seems that way considering PaymentSheet (and CustomerSheet) use `deviceSupportsApplePay`, which hooks into `self.supportedPKPaymentNetworks()`, which includes `additionalEnabledApplePayNetworks`.

cc: @davidme-stripe

Reply:
Correct!